python/llm/example/CPU/HF-Transformers-AutoModels/Model/distil-whisper/long-segment-recognize.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
```

Comment:
Can we merge short and long segment into the same py file? And use a parameter, e.g. '--long-seg' to control whether to use the long seg path? 
Then in README, you can merge the install & run part (remember to explain the extra --long-seq, batch size, chunk-len params). In sample output you can show 1 example output w/ long-seq param and 1 example w/o long-seq param

Reply:
Btw, if long seg can also handle short sequences well (e.g. <30s) We can use long-seq=True as default.